### ISSUE TITLE
Bug Report: iBooks Table of Contents not populating correctly.

### ISSUE THREAD

**Issue by Novel-Idea** [daniel-j/fimfic2epub#6]
The Table of Contents are not showing up properly in iBooks. While the HTML version in the book itself works fine, you can see that the page numbers in the screenshot below are not populating and nothing happens when you click on them.

![screen shot 2016-08-18 at 9 17 06 am](https://cloud.githubusercontent.com/assets/21106478/17781603/c536a59c-6524-11e6-8d15-66cad6644e68.png)

**Comment by daniel-j** [comment]
If you wait a little while, iBooks will scan the book and count pages. The dots indicate that. When that's done, you have page numbers! Clicking items in the iBooks TOC works too.
![screen shot 2016-08-18 at 19 22 38](https://cloud.githubusercontent.com/assets/1264014/17783619/21aaae0a-6579-11e6-8f50-a5a30be74e61.png)

**Comment by Novel-Idea** [comment]
Something very screwy is going on then because I just tried this with my own _Applications_ story. I restarted the program, let it sit in the background for 5 minutes, closed and reopened the book a few times. That data is still not populating.

In addition, when trying to use the Table of Contents in iOS, using the iOS interface simply takes me to the front of the book. 

![screen shot 2016-08-18 at 10 31 22 am](https://cloud.githubusercontent.com/assets/21106478/17783941/44b42b50-652f-11e6-8dd6-38313b16d42e.png)
